doc/classes/EditorScript.xml

```diff
@@ -1,10 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorScript" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Simple script to perform changes in the currently edited scene.
+		Base script that can be used to add extension functions to the editor.
 	</brief_description>
 	<description>
-		This script can be run from the Scene -&gt; Run Script menu option.
+		Scripts extending this class and implementing it's [code]_run()[/code] method can be executed from the Script Editor's [code]File -> Run[/code] menu option (or by pressing [code]CTRL+Shift+X[/code]) while the editor is running. This is useful adding custom in-editor functionality to Godot. For more complex additions, consider using [EditorPlugin]\ s instead. Note that extending scripts need to have [code]tool mode[/code] enabled.
```

Comment:
Grammar: 
- "its", not "it's"
- This is useful **for** adding custom in-editor functionality to Godot
- "using [EditorPlugin]\ **s** instead" : one "s" alone
- (not sure) "Note that extending scripts need**s** to have..."

Reply:
Fixed the first two, "[EditorPlugin]\ s" should become "[EditorPlugin]s" once parsed, and the last one should be correct as is.